src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/locationfinder/AppointmentLocationsService.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.locationfinder
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingRepository
+import uk.gov.justice.digital.hmpps.whereabouts.services.PrisonApiService
+
+@Service
+class AppointmentLocationsService(
+  private val prisonApiService: PrisonApiService,
```

Comment:
Contains all the data fetching and munging code.  The algorithm is provided by AppointmentLocationsFinder which is called from this class.  I'm sure there's a better way to structure this.

Reply:
Is this comment out of date?